Demo/Sources/Assets/Icons.xcassets/MarketIcons/finn/airplane.imageset/Contents.json

```diff
@@ -1,7 +1,7 @@
 {
   "images" : [
     {
-      "filename" : "Bilabonnement.svg",
```

Comment:
is this expected, at first glance it seems that it might not be ?

Reply:
now all icons are pdf